Jenkinsfile

```diff
@@ -92,7 +92,7 @@ node(platform) {
 
     stage("Archive") {
         if (platform == 'android') {
-            archiveArtifacts artifacts: "platforms/android/build/outputs/apk/android-${BUILD_CONFIG}.apk", excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
+            archiveArtifacts artifacts: "platforms/android/app/build/outputs/apk/${BUILD_CONFIG}/app-${BUILD_CONFIG}.apk", excludes: "platforms/android/build/outputs/apk/${BUILD_CONFIG}/*-unaligned.apk"
```

Comment:
Maybe this should just be `platforms/android/**/app-${BUILD_CONFIG}.apk` (and similar for the `excludes` path)? That might isolate it from future breaking path changes. Thoughts?

Reply:
I don't think we need the "excludes" param in this case since we are archiving a specific path rather than using wildcards.

Reply:
That's true...I'll remove that, then merge. Thanks for the review!